src/content/docs/style-guide/formatting/hyperlinks.mdx

```diff
@@ -105,6 +105,34 @@ When creating or editing an HTML `id` for an anchor link, follow these guideline
 * When reorganizing a doc, try to preserve incoming anchor links where possible. Be mindful that when you delete a level two heading `##`, it will break any links to that heading.
 * Avoid changing an existing anchor ID, because we may be monitoring traffic to it from our internal New Relic account.
 
+## Deep links into product [#deep-links]
+
+When directing users to parts of the product, you want to deep link when possible. Deep linking encourages users to go into the product and, if they haven't created an account yet, prompts them to sign up. This is more effective than just linking to a sign up page. However, deep linking into the UI only works sometimes and it takes some experimenting to find a good permalink. 
```

Comment:
@akristen Would it be helpful to define what a deeplink is and provide an example link?

Reply:
@akristen  +1 to add an example